stock_quant_manual_assign/models/stock_picking_type.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+class StockPickingType(models.Model):
```

Comment:
Flake8 is complaining there should be one more blank line here.
`stock_quant_manual_assign/models/stock_picking_type.py:6:1: E302 expected 2 blank lines, found 1`
```suggestion

class StockPickingType(models.Model):
```

Reply:
@TimLai125 Fixed it!